src/natcap/invest/urban_nature_access.py

```diff
@@ -78,6 +78,8 @@
                 'search_radius_m': {
                     'type': 'number',
                     'units': u.meter,
+                    'required':
+                        f'search_radius_mode == {RADIUS_OPT_GREENSPACE}',
```

Comment:
Thanks for catching this!  Yes, should be a string comparison.  Patched in e514fba35

Reply:
Oh wait, I see now that conditional requirement isn't implemented for fieldnames, but it also isn't breaking anything because only an explicit `True` evaluates to `True`.

Reply:
I'm going to look at this in https://github.com/natcap/invest/issues/1165